.dockerignore

```diff
@@ -0,0 +1,4 @@
+.git
+.gitignore
+tests
+action.yml
```

Comment:
You could easily add `.github`, `LICENCE` and `README.md` to this list since we don't need them too.